pages/common/git-extras.md

```diff
@@ -0,0 +1,16 @@
+# git extras
+
+> Set of commands for `Git`.
+> More information: <https://github.com/tj/git-extras>.
+
+- Download and or upgrade `git-extras` commands:
```

Comment:
This is a little bit confusing although I know what you mean.  This is less so but still covers the same breadth.
```suggestion
- Update `git-extras`:
```

Reply:
With the ubuntu repo, which is what most are probably using, the `git-extras` package doesn't come with any commands to my knowledge, so its probably worth leaving that it downs the commands

Reply:
Good point — let's skip this and apply [this](https://github.com/tldr-pages/tldr/pull/6037#discussion_r637580876) instead.